server/plugin/api_test.go

```diff
@@ -1065,7 +1065,7 @@ func TestCreateSubscription(t *testing.T) {
 				"user_id": "%s",
 				"type": "%s",
 				"channel_id": "%s"
-			  	}`, testutils.GetID(), constants.SubscriptionTypeRecord, testutils.GetChannelID()),
+			  	}`, testutils.GetID(), testutils.GetRecordTypeSubscription(), testutils.GetChannelID()),
```

Comment:
I meant, you should create a testutil function for getting the whole request body, not just the subscription type.